bin/build_cmd.ml

```diff
@@ -82,12 +82,40 @@ let run_build_system ~common ~request =
       Fiber.return ())
 ;;
 
-let run_build_command_poll_eager ~(common : Common.t) ~config ~request : unit =
+let lock_ocamlformat () =
+  if Lazy.force Lock_dev_tool.is_enabled
+  then
+    (* Note that generating the ocamlformat lockdir here means
+             that it will be created when a user runs `dune fmt` but not
```

Comment:
nitpick: can you fix the indentation here so the comment text lines up on each line